### ISSUE TITLE
Add peat block to evolution items in /shop 

### ISSUE THREAD

**Issue by Dxhcky** [SmashMC-Development/Suggestions#1030]
**Describe the suggestion**
As you can buy other evolution items, I think it makes sense to add the peat block as it is required to evolve an ursaring into an ursaluna. 

**Which servers does this apply to?**
All

**Additional context**
N/A

**Comment by SimaoAntunes26** [comment]
I agree with this, but they could also put some sweets there for milcery, just thought about it and maybe they could add incenses to.
